client/src/pages/cart.jsx

```diff
@@ -0,0 +1,189 @@
+import Header from '../components/Header';
+import NavBar from '../components/NavBar';
+import styled from 'styled-components';
+
+function Cart({ cartItems, onAdd, onRemove, onDelete }) {
+  const itemsPrice = cartItems.reduce(
+    (totalPrice, itemPrice) => totalPrice + itemPrice.price * itemPrice.qty,
+    0
+  );
+  const shippingPrice = itemsPrice > 20 ? 0 : 2.99;
```

Comment:
Does this work even if I delete items from my cart and fall below 20?

Reply:
The shipping price applies only if the itemPrice is lower than 20. Works perfectly as it should. ;)